src/constraint/dart/CollisionFree.cpp

```diff
@@ -35,15 +35,22 @@ statespace::ConstStateSpacePtr CollisionFree::getStateSpace() const
 //==============================================================================
 bool CollisionFree::isSatisfied(
     const aikido::statespace::StateSpace::State* _state,
-    TestableOutcome* outcome) const
+    __attribute__((unused)) TestableOutcome* outcome) const
```

Comment:
Should this just be deleted (or at least the parameter name, if we're not using it)?